harc_game_web/apps/bank/models.py

```diff
@@ -26,7 +26,9 @@ class Bank(models.Model):
     date_accrued = models.DateTimeField(default=timezone.now)
     documented_task = models.ForeignKey(DocumentedTask, on_delete=models.RESTRICT, null=True, default=None)
     accrual = models.IntegerField(default=0, null=True)
```

Comment:
niektóre zadania chyba nie mają nagrody normalnej tylko samą specjalną - to tutaj też powinno być `blank=True` itd. bo to pole nie jest wymagane

Reply:
OK, ale to jest accrual, który w takim przypadku jest ustawiany na 0.